include/base_interface.h

```diff
@@ -298,13 +298,49 @@ class BaseInterface : public ParameterAcceptor, public SimulatorAccess<dim,space
   const std::vector<unsigned int> get_differential_blocks() const;
 
   /**
-   * This function is called in order to know which Mapping should be
-   * used.  By default it returns
-   * dealii::StaticMappingQ1<dim,spacedim>::mapping.
-   * If you want to use different mapping you need to overwrite this
-   * function.
+   * Set the mapping used where no different mapping has been specified.
+   * Return dealii::StaticMappingQ1<dim,spacedim>::mapping by default.
    */
-  virtual const Mapping<dim,spacedim> &get_mapping() const;
+  virtual const Mapping<dim,spacedim> &get_default_mapping() const;
+
+  /**
+   * Set and return the mapping used for the output.
```

Comment:
This is not coherent. What you do internally is of no concern to the user. This is a getter, not a setter. So you don't `Set and return`, but simply `Return the mapping to use for the output.`